viz/components/dust/slideshow/v1/index.tsx

```diff
@@ -321,12 +309,12 @@ function validateSlideChildren(
       return true;
     }
 
-    return !validSlideDisplayNames.has(child.type.displayName);
+    return false;
   });
 
   if (invalidChildren.length > 0) {
     throw new Error(
-      "Slideshow: All children must be Slideshow.Preset components. " +
+      "Slideshow: All children must be valid React components with display name. " +
```

Comment:
Loosen the rule a bit to make it customizable

Reply:
We should probably not make it too strict but at least default to Slideshow.Preset. Can be done later.

Reply:
The problem is I need to have `SlideshowRoot` for customized templates too (which we would like to support) for navigation and preview, but if we have this check you cannot really have customized templates 🙃 I need to think more how we can support both cases though. Let's see how it does without it!